Source/Modules/javascript.cxx

```diff
@@ -469,12 +575,59 @@ int JAVASCRIPT::classHandler(Node *n) {
   emitter->switchNamespace(n);
 
   emitter->enterClass(n);
-  Language::classHandler(n);
+  int returnValue = TypeScriptTypes::classHandler(n);
   emitter->exitClass(n);
 
+  return returnValue;
+}
+
+/**
+ * Executed by SWIG to manage a declared C++ public variable
+ *
+ * @param n The node representing the public variable
+ * @return status to swig decide what to do
+ */
+int JAVASCRIPT::membervariableHandler(Node *n)
+{
+  return TypeScriptTypes::membervariableHandler(n);
+}
+
+/**
+ * Executed for each enum, found when navigating in the node tree
+ *
+ * @param n The node that represents the enumerated
+ * @return The SWIG status
+ */
+int JAVASCRIPT::enumDeclaration(Node *n)
+{
+  TypeScriptTypes::enumDeclaration(n);
   return SWIG_OK;
 }
 
+/**
+ * Executed when navigating in the tree, when a value declaration node
+ * is found.
+ *
+ * @param n The node of the value declaration
+ * @return The SWIG status
+ */
+int JAVASCRIPT::enumvalueDeclaration(Node *n)
+{
+  return TypeScriptTypes::enumvalueDeclaration(n);
+}
+
+/**
+ * Handler executed in tree navigation when it encounters a class
+ * member function
+ *
+ * @param n The node that represents class member function
+ * @return SWIG status
+ */
+int JAVASCRIPT::memberfunctionHandler(Node *n)
+{
+  return TypeScriptTypes::memberfunctionHandler(n);
+}
+
```

Comment:
Sorry, I don't understand what's going on here. AFAICS all this should be just removed, why should we define methods in the derived class just calling their base class equivalents? This seems to be exactly the same as not declaring (nor defining) them at all.